### ISSUE TITLE
Developer greyed options for scenario selection isn't clear on permissions

### ISSUE THREAD

**Issue by Lextuga007** [The-Strategy-Unit/nhp_inputs_selection_app#33]
From QA and getting a test scenario set up for nhp_outputs there was confusion with the greyed out options for the scenario options as they suggested "no permissions", but really meant "no previous scenario was available". 

This came about particularly because there were multiple sites available from the developer view and although nhp_inputs wasn't being tested it would be useful to test users have correct restrictions.

**Comment by tomjemmett** [comment]
## radio buttons

currently we choose whether to enable or disable these based on a condition (is the length of the existing scenario files > 0). We could switch this to instead change the checkbox options: if the length is > 0 then show all 3, else just show the first option.

## validation messages

the scenario name validation error message isn't the only message that can appear there, for example:

<img width="1237" height="373" alt="Image" src="https://github.com/user-attachments/assets/52ddf5b4-44c9-4ec9-a301-d90adf188212" />

it's using shiny validation inside of a reactive, any UI that consume that reactive will show the validation messages if present, and will fail to continue to render. that's why these have been tied to the start button (if validation fails then the button won't appear).

so there are two issues here:

1. moving the text above the radio buttons would need a new UI element to be created. We would then also need to handle the start button differently, otherwise we will show the same error messages twice
2. the colour of the text is controlled by the shiny.scss file with the `$shiny-error-validated-color` sass variable, which is set to `#888` (the light grey). we could easily override this

**Comment by tomjemmett** [comment]
example of how to change the colour of the error message, at the end of `app_ui.R` replace the ui_theme with:

``` r
ui_theme <- function() {
  brand <- brand.yml::read_brand_yml(app_sys("_brand.yml"))

  bslib::bs_theme(brand = brand) |>
    bslib::bs_add_rules(
      sass::sass_file(app_sys("www/styles.scss"))
    ) |>
    bslib::bs_add_variables(
      "shiny-error-validated-color" = "#DA291C"
    )
}
```

example colour there is the NHS E theme's "Emergency Red" colour.
